shared/chat/conversation/container.js

```diff
@@ -87,7 +90,10 @@ export default compose(
     onEditLastMessage: props => () => props.setEditLastMessageCounter(props.editLastMessageCounter + 1),
     onFocusInput: props => () => props.setFocusInputCounter(props.focusInputCounter + 1),
     onScrollDown: props => () => props.setListScrollDownCounter(props.listScrollDownCounter + 1),
-    onToggleSidePanel: props => () => props.setSidePanelOpen(!props.sidePanelOpen),
+    onToggleSidePanel: props => () => {
```

Comment:
close the keyboard when you open this. Its possible this could work automatically w/ route changes but this isn't really using the route mechanism. KISS for now